.mailmap

```diff
@@ -159,4 +164,19 @@ Xavier Bouthillier <xavier.bouthillier@gmail.com> Xavier Bouthillier <xavier.bou
 Xavier Bouthillier <xavier.bouthillier@gmail.com> Xavier Bouthillier/ <xavier.bouthillier@gmail.com>
 Orhan Firat <orhan.firat@ceng.metu.edu.tr> orhanf <orhan.firat@ceng.metu.edu.tr>
 Søren Kaae Sønderby <skaaesonderby@gmail.com> skaae <skaaesonderby@gmail.com>
-Luke Metz <luke.metz@students.olin.edu> = <luke.metz@students.olin.edu>
\ No newline at end of file
+Luke Metz <luke.metz@students.olin.edu> = <luke.metz@students.olin.edu>
+Guillaume Alain <guillaume.alain.umontreal@gmail.com> Guillaume Alain <gyomalin@gmail.com>
```

Comment:
All that is good. But I think next time we should sort it alphabetically. Otherwise, it is starting to be hard to know if someone is in this list or not.